### PR TITLE
Package maintenance

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

**Comment by lundibundi** [comment]
Why was the windows dropped though?

Also if this is going to be released as 0.1.3 then we shouldn't remove Node.js 8 from CI, otherwise (to release 0.2.0) we also have to change `package.json/engines` field to `>=10`

File: package.json [hunk 1]
```diff
@@ -30,12 +30,12 @@
     "node": ">=8.6.0"
```
Comment (lundibundi) [inline package.json:30]:
```suggestion
    "node": ">=10.0.0"
```

**Comment by lundibundi** [comment]
Also, all commits can be squashed into 1 as `Drop windows builds in CI` is no longer needed and all fixes are present in `fixup! Drop windows builds in CI`

**Comment by tshemsedinov** [comment]
Landed in https://github.com/metarhia/metastreams/commit/17beb3b3303269c7825ff00afa7a2a631547b617, https://github.com/metarhia/metastreams/commit/efe452f416c1fbb4417a67fb2b4bbb5c09cdbc84 and https://github.com/metarhia/metastreams/commit/ec95397868229e41dfce03a98337ba59c7ca506b

**Pull request closed without merging**